docs/Installation.rst

```diff
@@ -178,25 +121,142 @@ The following command should now mount the cgroup filesystem:
+
+To "activate" it:
 
-If you prefer using Python Package Index, you can retrieve all Python dependencies with this line:
+.. sourcecode:: bash
+
+    . ~/cms_venv/bin/activate
+
+After the activation, python commands (python, pip) will refer to their corresponding virtual version, so you can install python dependencies by issuing:
+
+.. sourcecode:: bash
+
+    pip install -r requirements.txt
+
+.. note::
+
+    Once you finished installing CMS (and using it) you can deactivate the virtual environment by issuing:
+
+    .. sourcecode:: bash
+
+        deactivate
+
+Ubuntu
+------
+
+.. warning::
+
+  It's usually possible to install python dependencies using your Linux distribution's package manager. However, keep in mind that the version of each package is controlled by the package mantainers and could be too new or too old for CMS. This is generally not the case on Ubuntu (since the package versions are usually stable and quite old).
+
+To install CMS python dependencies on Ubuntu, you can issue:
+
+.. sourcecode:: bash
+
+    sudo apt-get install python-setuptools python-tornado python-psycopg2 \
+         python-sqlalchemy python-psutil python-netifaces python-crypto \
+         python-tz python-six python-beautifulsoup python-mechanize \
+         python-coverage python-mock python-requests python-werkzeug \
+         python-gevent patool
+
+    # Optional.
+    # sudo apt-get install python-yaml python-sphinx python-cups python-pypdf2
+
+Arch Linux
+----------
+
+.. warning::
+
+  It's usually possible to install python dependencies using your Linux distribution's package manager. However, keep in mind that the version of each package is controlled by the package mantainers and could be too new or too old for CMS. This is especially true for Arch Linux, which is a bleeding edge distribution.
+
+To install CMS python dependencies on Arch Linux (again: assuming you didn't use the aforementioned AUR packages), you can issue:
 
 .. sourcecode:: bash
 
-    sudo pip install -r requirements.txt
+    sudo pacman -S python2-setuptools python2-tornado python2-psycopg2 \
+         python2-sqlalchemy python2-psutil python2-netifaces python2-crypto \
+         python2-pytz python2-six python2-beautifulsoup3 python2-mechanize \
+         python2-mock python2-requests python2-werkzeug python2-gevent \
+         python2-coverage
 
+    # Install the following from AUR.
+    # https://aur.archlinux.org/packages/patool/
+
+    # Optional.
+    # sudo pacman -S python2-yaml python-sphinx python2-pycups
+    # Optionally install the following from AUR.
+    # https://aur.archlinux.org/packages/python2-pypdf2/
 
 Installing CMS
 ==============
 
+In order to run CMS there are some preparation steps to run (like installing the sandbox, compiling localization files, creating the ``cmsuser``, and so on). You can either do all these steps by hand or you can run the following command:
+
+.. sourcecode:: bash
+
+    sudo ./prerequisites.py install
```

Comment:
FYI: the localization files are picked up also locally, no need to install them. But I think it is better to install everything anyway.